examples/visibility_ingest_cookbook/config_winchefclient/spec/unit/recipes/default_spec.rb

```diff
@@ -0,0 +1,11 @@
+#
+# Cookbook Name:: config_winchefclient
+# Spec:: default
+#
+# Copyright 2016, All Rights Reserved.
```

Comment:
Can you remove all proprietary copyright header? This conflicts with our cookbook license